src/test/java/io/fabric8/maven/docker/config/ConfigHelperTest.java

```diff
@@ -71,11 +69,41 @@ public void externalPropertyActivation() throws MojoFailureException {
             ConfigHelper.validateExternalPropertyActivation(project, images);
             fail();
         }catch(MojoFailureException ex) {
-            assertTrue(ex.getMessage().contains("Configuration error: Cannot use property"));
+            assertEquals("Configuration error: Cannot use property " + ConfigHelper.EXTERNALCONFIG_ACTIVATION_PROPERTY + " on projects with multiple images without explicit image external configuration.",
```

Comment:
I would only check on parts with a essential to provide a meaningful context in the error message. That way the check is more robust on changes wrt rewording of the message. So, I always prefer contains on the essential parts (like `contains(ConfigHelper.EXTERNALCONFIG_ACTIVATION_PROPERTY)`)